src/test/java/uk/gov/hmcts/reform/pip/data/management/service/ValidationServiceTest.java

```diff
@@ -297,14 +296,12 @@ void testExceptionWhenValidatingPayload() {
     @Test
     void testValidateWithErrorsWhenArtefactIsDailyCauseList() throws IOException {
         try (InputStream jsonInput = this.getClass().getClassLoader()
-            .getResourceAsStream("mocks/dailyCauseListInvalid.json")) {
+            .getResourceAsStream("mocks/civil-daily-cause/dailyCauseListInvalid.json")) {
```

Comment:
naming conventions here are different to the int tests, its tiny but keeping consistent naming will help the others who havent been as involved as you and partly me with the many different list types,

Reply:
Made this consistant with int tests